__tests__/app/message-table.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from "@testing-library/react";
+import { MessageTable } from "@/app/message-table";
+import { experimentColumns, RecipeOrBranchInfo } from "@/app/columns";
+import { ExperimentFakes } from "@/__tests__/ExperimentFakes.mjs";
+import { NimbusRecipe } from "@/lib/nimbusRecipe";
+
+describe("MessageTable", () => {
+  describe("ExperimentColumns", () => {
+    it("renders the branch description and slug", () => {
+      const rawRecipe = ExperimentFakes.recipe("test-recipe");
+      const nimbusRecipe = new NimbusRecipe(rawRecipe);
+      const messageTableData: RecipeOrBranchInfo[] = nimbusRecipe
+        .getRecipeOrBranchInfos()
+        .flat(1);
```

Comment:
The `.flat(1)` isn't necessary here.  Please remove it.

The fact that we have/need it in the non-test code is a bug.  Please add an XXX comment to page.tsx making this clear.